languages/states.php

```diff
@@ -2262,17 +2262,15 @@
 		'AG' => __( 'Agrigento', 'lifterlms' ),
 		'AL' => __( 'Alessandria', 'lifterlms' ),
 		'AN' => __( 'Ancona', 'lifterlms' ),
-		// 'Aosta' manually added, see https://github.com/gocodebox/lifterlms/issues/2098.
 		'AO' => __( 'Aosta', 'lifterlms' ),
 		'AP' => __( 'Ascoli Piceno', 'lifterlms' ),
-		// 'Arezzo' manually added, see https://github.com/gocodebox/lifterlms/issues/2214.
 		'AR' => __( 'Arezzo', 'lifterlms' ),
 		'AT' => __( 'Asti', 'lifterlms' ),
 		'AV' => __( 'Avellino', 'lifterlms' ),
 		'BA' => __( 'Bari', 'lifterlms' ),
 		'BT' => __( 'Barletta-Andria-Trani', 'lifterlms' ),
 		'BL' => __( 'Belluno', 'lifterlms' ),
-		'BN' => __( 'Benevento Province', 'lifterlms' ),
+		'BN' => __( 'Benevento', 'lifterlms' ),
```

Comment:
Can you add a comment like the one above about Arezzo? because this file was automatically generated, so let's keep track of the changes in a way.

Reply:
Got it, all changes are done!